config.lua.dist

```diff
@@ -98,15 +98,21 @@ serverSaveShutdown = true
 
 -- Rates
 -- NOTE: rateExp is not used if you have enabled stages in data/XML/stages.xml
+-- rateSpawn limits the number of monsters that will spawn within the same spawn object
 rateExp = 5
 rateSkill = 3
 rateLoot = 2
 rateMagic = 3
 rateSpawn = 1
 
 -- Monsters
+-- spawnIntervalRate controls the rate of monster spawning interval
+-- its in percentage so 100 = 100% (normal spawn), 50 = 50% (half spawn time) and so on
+-- the higher the number, the spawn will be less frequent
+-- NOTE: minimum spawn interval is 10 seconds while maximum is 1 day for the map spawn file
```

Comment:
User is setting the intervalRate in % but the limits are in seconds/days (time). This seems confusing and if the resulting interval falls out of bounds, the monster will not spawn at all.
I think that most users will not know all the intervals in `spawns.xml` and a better default behavior would be to cap the interval and spawn the monster with the lowest/highest permissible interval instead to avoid confusion with the monster simply not spawning at all.

Reply:
What you would suggest, adding or subtracting absolute values?

I would suggest to just ditch this feature, doesn't make much sense, nor has much use. It's just to compensate the fact people thought they had it when they actually didn't, but never truly missed it anyway. Since it's blocking 1.4 I would only add the ```-- rateSpawn limits the number of monsters that will spawn within the same spawn object``` line to config.lua and scrap the idea.

Reply:
Using this 100 == 1 and 200 == 2, so it would be extremely difficult to go outside of bounds here. Also, as the current implementation goes, there is nothing stopping a player from being able to "go outside of bounds" in the spawns.xml anyways, pretty much same thing, just relocated and / 100. I see no reason this should stall 1.4